src/errors/transactionErrors.ts

```diff
@@ -47,6 +55,12 @@ export const checkForTokenTransactionError = (receipt: TransactionReceipt) =>
 export const checkForVaiControllerTransactionError = (receipt: TransactionReceipt) =>
   checkForTransactionError(
     receipt,
-    VAIControllerErrorReporterError,
-    VAIControllerErrorReporterFailureInfo,
+    VaiControllerErrorReporterError,
+    VaiControllerErrorReporterFailureInfo,
   );
+
+export const checkForVaiVaultTransactionError = (receipt: TransactionReceipt) =>
+  checkForTransactionError(receipt, VaiVaultErrorReporterError, VaiVaultErrorReporterInfo);
+
+export const checkForXvsVaultTransactionError = (receipt: TransactionReceipt) =>
```

Comment:
This is actually for checking the vaultProxyContract because the Vault Contract doesn't send these events